src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -46,7 +47,7 @@ public class Hardware {
 							Units.inchesToMeters(30)),
 					// Camera has a slight yaw, around -0.25 radians following right-hand rule (thumb points
 					// to +Z/up, fingers curl in positive rotation)
-					new Rotation3d(0, 0, -0.25));
+					new Rotation3d(0, 0, Math.toRadians(cameraYawDegrees)));
```

Comment:
-12 degrees is not the same as -0.25 radians, did you mean to change the value?

Reply:
-0.25 was estimated in Desmos and -12 degrees was from a protractor.